### PR TITLE
Adds CNR 1.5.9 release notes

### PR DESCRIPTION
**Pull request by Petahhh**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

None

### CODE REVIEW

File: release-notes.hbs.md [hunk 1]
```diff
@@ -29,7 +29,10 @@ The following issues, listed by component and area, are resolved in this release
 #### <a id='1-5-9-COMPONENT-NAME-ri'></a> v1.5.9 Resolved issues: COMPONENT-NAME
  
 - Resolved issue description.
- 
+
+#### <a id='1-5-9-cnr-ri'></a> v1.5.9 Resolved issues: Cloud Native Runtimes
+
+- Cloud Native Runtimes was rebuilt with an updated golang base image with version 1.20.12 to address CVEs
```
Comment (andrew-su) [inline release-notes.hbs.md:35]:
```suggestion
- Cloud Native Runtimes was rebuilt with an updated golang base image (version 1.20.12) to address CVEs
```

**Comment by krsna-m** [comment]
lgtm

**Comment by andrew-su** [comment]
lgtm

**Comment by Petahhh** [comment]
closing as it looks like the CVEs landed in the release notes already

**Pull request closed without merging**